src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.cs

```diff
@@ -133,5 +133,9 @@ internal static bool IsPrimitiveType(this CorElementType et)
         [Intrinsic]
         internal static bool IsKnownConstant<T>(T t) where T : struct => false;
 #pragma warning restore IDE0060
+
+        /// <returns>true if the given type is reference type, value type that contains references, or ByRefLike with references or by-refs; otherwise, false.</returns>
```

Comment:
```suggestion
        /// <returns>true if the given type is reference type, value type that contains references or by-refs; otherwise, false.</returns>
```